#! /usr/bin/env python

PACKAGE='compressed_depth_image_transport'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
  
gen.add("depth_max", double_t, 0, "Maximum depth value (meter) ", 10 , 1, 100)
gen.add("depth_quantization", double_t, 0, "Depth value at which the sensor accuracy is 1 m (Kinect: >75)", 100, 1, 150)
gen.add("png_level", int_t, 0, "PNG compression level", 9, 1, 9)

 
exit(gen.generate(PACKAGE, "CompressedDepthPublisher", "CompressedDepthPublisher"))
